includes/wsuwp-shortcode-fundselector-utils.js

```diff
@@ -265,9 +265,15 @@ var EPSILON = 0;
 				};
 				
 				jQuery("#advFeeAmount").text(_.template(wpData.adv_fee_message)({advFeeAmount: advFeeDecimal.toFixed(2) }));
+				jQuery("#advFeeAmount").html(jQuery("#advFeeAmount").html().replace("Advancement Fee", '<a href="https://foundation.wsu.edu/about/fee-disclosure/" target="_blank">Advancement Fee</a>'));
```

Comment:
This should also include the `rel="noopener noreferrer"` attribute in addition to `target="_blank"` for both [security](https://github.com/rwjblue/ember-template-lint#link-rel-noopener) and [performance](https://jakearchibald.com/2016/performance-benefits-of-rel-noopener/) reasons.